packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -131,7 +131,7 @@ const LRUSegmentComparer: Comparer<LRUSegment> = {
 interface IReferenceSearchInfo {
 	mergeTree: MergeTree;
 	tileLabel: string;
-	tilePrecedesPos?: boolean;
+	forwards?: boolean;
```

Comment:
i would leave this untouched. No need to change it now that the new function doesn't use `IReferenceSearchInfo`.